main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1597,21 +1599,45 @@ private static void getExtraOnlineInfo(@NonNull final Geocache cache, final Stri
         final String userToken = parseUserToken(page);
         final Observable<LogEntry> logs = getLogs(userToken, Logs.ALL);
         final Observable<LogEntry> ownLogs = getLogs(userToken, Logs.OWN).cache();
-        final Observable<LogEntry> specialLogs = Settings.isFriendLogsWanted() ?
-                Observable.merge(getLogs(userToken, Logs.FRIENDS), ownLogs) : Observable.empty();
-        final Single<List<LogEntry>> mergedLogs = Single.zip(logs.toList(), specialLogs.toList(),
-                (logEntries, specialLogEntries) -> {
+        final Observable<LogEntry> friendLogs = Settings.isFriendLogsWanted() ?
+                getLogs(userToken, Logs.FRIENDS).cache() : Observable.empty();
+
+        final List<LogEntry> ownLogsFromDb;
+        if (!ownLogs.isEmpty().blockingGet()) {
+            ownLogsFromDb = DataStore.loadLogsOfAuthor(cache.getGeocode(), GCConnector.getInstance().getUserName(), true);
```

Comment:
Note to myself: Check implementation with deactivated friend-option

Reply:
Works
* applyCommonsToOwnLog is called on own logs, where the friend-flag is set
* if for what ever reason, the friend-flag is not set in the db, we first read the entries with active flag, and if there are no such entries, we read the entries without active flag